ML_tutorial.ipynb

```diff
@@ -0,0 +1,7293 @@
+    "\n",
+    "We will work with *numerical*, *categorical*, and *text* features:\n",
+    "\n",
+    "\n",
+    "1. Numerical features: features that are numbers\n",
+    "    1. Discrete: numerical values that are integers (age in years)\n",
+    "    2. Continuous: numerical values that can be any type of number (velocity)\n",
+    "    \n",
+    "2. Categorical features: feature value is chosen from a small group of possible options (college major)\n",
+    "\n",
+    "3. Text features: these features include strings that are extremely unique to a given instance (summary)\n",
+    "\n",
+    "The features tell us everything we know about the data we are predicting on. \n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Instances\n",
+    "\n",
+    "Each row represents an instance. An instance is a singular unit of data. Our goal is that the model will be able to predict the correct label given the features from the instance. In our example, each line of spoken dialogue is a unique instance and each instance has a value for each feature."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exploratory Data Analysis"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Exploratory Data Analysis is the process of analyzing your data in order to understand its main chatacteristics. It's like:\n",
+    "1. reading the ingredients before taking a bite of a mystery nugget. \n",
+    "2. running a background check before a blind date\n",
+    "\n",
+    "\n",
+    "It goes back to knowing what you are getting yourself into before you dive in"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Survey the Data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['Season_x',\n",
+       " 'Episode',\n",
+       " 'Sentence',\n",
+       " 'Name',\n",
+       " 'N_serie',\n",
+       " 'N_Season',\n",
+       " 'Emision Date',\n",
+       " 'Season_y',\n",
+       " 'Episode Number',\n",
+       " 'Number in Season',\n",
+       " 'Episode Name',\n",
+       " 'Director',\n",
+       " 'Writer',\n",
+       " 'Original Air Date',\n",
+       " 'US viewers (million)',\n",
+       " 'Runtime (mins)',\n",
+       " 'IMDB Description',\n",
+       " 'IMDB votes',\n",
+       " 'Imdb Rating',\n",
+       " 'Notable Death Count']"
+      ]
+     },
+     "execution_count": 32,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# What even are the features?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[62,\n",
```

Comment:
probably best if we remove the outputs from the code that was there, eh? Distracting otherwise?

Reply:
good call!
outputs cleared!